packages/relay/src/utils.ts

```diff
@@ -67,4 +67,42 @@ export class Utils {
   public static generateRequestId = (): string => {
     return crypto.randomUUID();
   };
+
+  /**
+   * Estimates the transaction fees for file create and file append transactions based on the provided
+   * call data size, file chunk size, and current network exchange rate.
+   *
+   * @param {number} callDataSize - The size of the call data in bytes.
+   * @param {number} fileChunkSize - The size of each file chunk in bytes.
+   * @param {number} currentNetworkExchangeRateInCents - The current network exchange rate in cents.
+   * @returns {number} The estimated transaction fee in tinybars.
+   */
+  public static estimateFileTransactionsFee(
+    callDataSize: number,
+    fileChunkSize: number,
+    currentNetworkExchangeRateInCents: number,
+  ): number {
+    const fileCreateTransactions = 1;
+    const fileCreateFeeInCents = constants.NETWORK_FEES_IN_CENTS.FILE_CREATE_PER_5_KB;
+
+    // The first chunk goes in with FileCreateTransaciton, the rest are FileAppendTransactions
+    const fileAppendTransactions = Math.floor(callDataSize / fileChunkSize) - 1;
```

Comment:
Perhaps I am missing something here, but I think we want to use `Math.ceil` here instead of `.floor` Using `.floor` will round down to the nearest integer, possibly missing the last chunk.  `.ceil` will round up to the nearest integer catching the last chunk even if the `callDataSize` is not an exact multiple of the `fileChunkSize`.

Reply:
I think this logic is outdated, @quiet-node can you verify